### DISCUSSION TITLE
Who's hiring? (Feb 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#33869] [Ideas]
# What is this?
This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #32937. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you're a developer looking to post about themselves, use #33870.

**Comment by yulijaflynn** [comment]
**[Witt-Gruppe](https://karriere.witt-gruppe.eu/berufsfelder/tech-development/) | (Senior) Front-End Developer | Remote (Germany only) | Full-time | [Link](https://karriere.witt-gruppe.eu/job/3412870-frontend-developer-m-w-d-remote-arbeit-moeglich/)**

Witt-Gruppe (a member of the Otto-Group) is hiring for a variety of positions, including a front-end developer to work on our Witt Commerce Cloud (WCC), which is an in-house developed platform for our web shops, that has been tailored to our needs.
Our platform is developed with state of the art technology, such as `React, Next.js, GraphQL and TypeScript`.

If you're interested or if you have any questions, please feel free to get in touch with me or apply directly via the link above.
I'm happy to see you soon in our team 🙂 

[Twitter](https://twitter.com/yuli_yulija) | [LinkedIn](https://www.linkedin.com/in/julia-w-ab90a41a1/) | [Xing](https://www.xing.com/profile/Julia_Woehrl) | [Mail](mailto:julia.woehrl@witt-gruppe.eu)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by spencermarx** [comment]
### [LaFabrica](https://www.lafabrica.work/) | Full-Stack Engineers | Remote (Worldwide) | Full-time & Part-Time | [Link](https://www.lafabrica.work/developers/register)

We are looking for Fullstack Engineers to join our platform, [LaFabrica](https://www.lafabrica.work/), a crypto-driven cooperative for Software Engineers.

**The platform...**
1. Connects developers across the world directly to remote software projects paid in US Dollars 💵
2. Offers members achievement-based dividends, distributed using our very own cryptocurrency 🏦
3. Provides startups and dev agencies dedicated, thoroughly vetted, ready-to-work remote engineering teams and individual developer resources within a range of budgets 👩‍💻👨‍💻

Our Beta pool is filling up quickly, so if you’re interested, sign up soon!

Registration: ➡️ [here](https://www.lafabrica.work/developers/register) ⬅️

**Our Stack:**
Completely flexible -- from full stack Javascript to GoLang, we encourage devs from any stack to apply.

The current platform is built out using:
* Typescript
* NextJS | React | Material UI
* NestJS | Firebase
* AWS Lambda | Google Cloud Functions

Feel free to apply using the link above or reach out to me directly.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by sb2nov** [comment]
[corise](https://corise.com/) | Full-Stack Engineer | Remote (US) | Full-time | $100-150k base | [Link](https://corise.notion.site/Full-Stack-Software-Engineer-9e7e92a5954b47eea42655ea3a220d6b)

We are a small but mighty team working on a mission to upskill the world's workforce.

Over the next decade, 1 billion jobs will be reimagined by the development of new technologies. Companies need to reskill their employees, but they lack the know-how to do so. Individuals need to reskill throughout their careers to keep pace with changing requirements, but their learning options are insufficient.

We are building a better solution using social learning and technology to drive outcomes at scale. Our project-based learning platform empowers employees to make step function transformations in their careers and companies to upskill their workforces.

Feel free to apply [here](https://corise.notion.site/Full-Stack-Software-Engineer-9e7e92a5954b47eea42655ea3a220d6b).

**Reply by sup-Udh** [reply]
i would love to work with you guys! @sb2nov applied!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by jaxomlotus** [comment]
BurnerPage | Front-End Engineer | NYC or Remote (US only) | Full-time | $115-175k base + benefits + generous equity | [Link](https://burnerpage.com/jobs/3/front-end-engineer)

[BurnerPage](https://burnerpage.com) is a stealth startup founded by a repeat entrepreneur who sold his last company to Adobe. We are VC backed and looking to make hires 2-5 as we bring our product to market. The above link is for [front-end devs](https://burnerpage.com/jobs/3/front-end-engineer), but we're looking for [backend](https://burnerpage.com/jobs/2/back-end-engineer) / fullstack and [machine learning engineers](https://burnerpage.com/jobs/1/machine-learning-engineer) as well (with different base pay for those positions depending on experience). 

BurnerPage is building the future of webpages, harnessing the power of AI and Machine Learning to automatically build webpages and tailor the viewer experience to the individual. [Note: The [current product](https://burnerpage.com/) does not yet advertise this purpose - it was just a manual prototype we launched that hit [#1 on ProductHunt](https://www.producthunt.com/posts/burner-page), but the stealth AI based product is the real focus that we're hiring for].I'm the founder and looking to find amazing folks who love working in Next.js. Feel free to reach out to me directly to apply at avi -(a)- burnerpage.com.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by Chris-Behling** [comment]
10 Pound Gorilla | Senior Full-Stack Engineer | Remote (US, Canada) | Contract | $30-$90/hr | [Link](https://www.indeed.com/viewjob?t=nextjs+react+developer+contractor+ecommerce+application&jk=dc45085e68d47c50&_ga=2.53118888.1263963683.1643646783-752049390.1643214935)

10 Pound Gorilla is building a multi-tenant eCommerce application which supports limited content management. We’re utilizing .NET APIs, NextJS, redeux, and serverless for CI/CD. 

Currently we have all of the HTML and CSS mark-up coded and ready to be integrated into Next/React components and pages. We have a couple developers working with the Next/React integration at this point. Nearly all of the APIs have been developed, we have 2 currently wrapping up development.

Our goal is to find additional developers who can help fill the shortage we have in react development and who can help us get the project through the finish line.

We have a mixture of areas where we are looking for support, from wiring up react components and pages, redux and state management, code deployment, payment gateway integrations, and identity provider integrations.

We are looking to hire the right candidates immediately and anticipate the project to be 3-6 months when factoring in the MVP launch and post-launch production support.

If this project sounds of interest and you feel you or your organization would be a good fit, we ask that you [check out our job posting on Indeed](https://www.indeed.com/viewjob?t=nextjs+react+developer+contractor+ecommerce+application&jk=dc45085e68d47c50&_ga=2.53118888.1263963683.1643646783-752049390.1643214935) and apply for the position. At that point we’d like to schedule an introduction call and to discuss the project and your team's capabilities in more detail.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by duosana** [comment]
Sana Benefits | Senior Full-Stack Engineer | Remote (US Timezone) | Full-time | $120K - 170K+ base + generous equity | [Link](https://jobs.lever.co/sanabenefits/943da474-5feb-4e67-bb3d-2a9be0f6b127?lever-via=1BLCQyk8VY)

We are looking to onboard several new mid/senior level full-stack engineering. We are trying to improve the health insurance industry by helping small businesses and medium sized companies. You can read more about us here: https://www.builtinaustin.com/2020/09/10/sana-benefits-raises-20m-series-a

We have a wonderful community of developers and a culture of growth and transparency. We enjoy supporting/mentoring each other and promoting from within. We offer competitive salaries and amazing benefits - we are a benefits company after all!

We are constantly adding new positions! You can check our all of our job listings [here](https://jobs.lever.co/sanabenefits?lever-via=1BLCQyk8VY).

For any questions, please reach out to me (#Dewo6376) or just comment your information below!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by juli1** [comment]
[Codiga](https://www.codiga.io/) | Frontend Engineer | Remote (everywhere) | Full-time | Comp based on location | [link to job posting](https://codiga.applytojob.com/apply/UTFca3mxVS/Frontend-Engineer)

We are a small startup that helps developers write better code faster by providing a Coding Assistant that lets you share and reuse snippets in your IDE and automate code reviews. Our team is fully remote with developers in the US and LATAM.

We are transitioning our frontend from a SPA application to NextJS and we are hiring an engineer skilled with NextJS. This is a full-time position and we hire anywhere. If you are up for a challenge (transition from React to Next) and willing to work with an international team, we would love to hear from you!

**Reply by juli1** [reply]
@JOSEPH-JENKINS - please **[apply](https://codiga.applytojob.com/apply/UTFca3mxVS/Frontend-Engineer)** and let's talk!

**Reply by Nutlope** [reply]
@juli1 Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

[a comment by an automated account is not shown]

**Comment by haydenbleasel** [comment]
### [Corellium | Senior Frontend Engineer | Remote (US, Canada or Australia) | Full-time](https://www.corellium.com/company/careers/4162717004-senior-frontend-engineer)

We’re on the lookout for a Senior Frontend Engineer to join the new Corellium Product Team, where we focus on the strategy and execution of our brand, product and community across the company and beyond. It’s a full-time, remote role focused on delivering stellar experiences for our website and product, as well as a range of upcoming projects.

I started the Product / Design team about 8 months ago and right now we're growing the team + working on a new design system, website, product frontend and lots more powered by the hottest frontend tech - Next, Vercel, Typescript, Rollup, etc. Fun times 😅

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by timlisiecki** [comment]
[Snyk](https://snyk.io/) | Frontend Engineer | Remote (US) | Full-time | [Link](https://boards.greenhouse.io/embed/job_app?token=5663456002&gh_src=3f9b65652us)
For this role, you will work as part of our Marketing Web Team within the Global Marketing Team to build a modern, static-generated site using Next.js and a headless CMS. You will partner with other developers on the development, optimization, and growth of our marketing websites.
I've been at the company for 6 months and love the team I'm working in. Our current website is Wordpress, but we are in the process of rebuilding it using a headless CMS with a Next.js frontend. Our team is in the Marketing Department, but we are made up of engineers and UX designers. Snyk is also[ growing rapidly](https://snyk.io/news/snyk-closes-530-million-series-f-investment-at-8-5-billion-valuation/), so it's a great time to join the team! Feel free to apply using the link above.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by omure** [comment]
Omure | Senior React.js Engineer | Remote (Europe, Americas) | Full-time | [Link](http://omure.com/jobs/senior-react) 

We're looking for experienced React/Next devs to join our Front-end team.

There's no legacy or client work involved and hours are flexible.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by valentinfily** [comment]
**BailFacile | Full-Stack Developer | Remote (France 🇫🇷) | Full-time | [Link](https://taleez.com/apply/fullstack-developer-php-js-bailfacile-cdi)**

We're a French Startup in the 🏠 property rental management space (SaaS) looking to hire a Fullstack developer (Laravel & Next.js). 
The position is 100% remote wherever you want in France 🇫🇷.
We are heavy users of Laravel and Next.js, and many more tools and technologies.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by jalada** [comment]
**[Pixie Labs](https://pixielabs.io) | Full Stack Developer (Senior Engineer) | Remote (UK-based) | Full-time | See job spec for compensation details | [Link](https://apply.workable.com/pixie-labs/j/18B26E12B9/)**

Pixie Labs is a digital product agency. We partner with organisations to design, build and launch software that brings their most valuable technology-driven experiences to life. As a senior engineer, you’ll help head up our clients’ development projects and write high-quality, reusable code whilst also sharing your knowledge and expertise with the rest of the team to help them do the same.

You may think an agency isn't for you - often it means hitting unrealistic deadlines imposed by clients, or writing terrible throwaway code just so the company gets paid. We're not like that; we focus on delivering at a sustainable pace and working closely with our clients to manage expectations. You can see for yourself how we manage and deliver projects in our [handbook](https://handbook.pixielabs.io/#/).

We're fairly polyglot, but specialise in and make heavy use of Ruby on Rails, Turbo (Hotwire and Stimulus), React and of course Next.js.

**Reply by originalpete** [reply]
Fantastic company – this is an incredible opportunity to work on a ton of interesting projects, learn a lot of tools, and make a big impact.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by delagroove** [comment]
## Mailchimp | Software Engineer | Remote (US-CA based) | Full time | 140k+

We are looking for passionate developers with experience in Javascript to join the Mailchimp Open Commerce Team. Our stack involves Node, React, GraphQl & Docker.

[Apply here!](https://mailchimp.com/jobs/listings/2587561-software-engineer-iii/) make sure you mention the NextJS board when applying!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by Charlette123** [comment]
Genie AI | Frontend Developer (React) | London OR UK Remote | Full-time | [https://apply.workable.com/genie-ai/j/8166DCED3C/](url)

Genie AI is a deep learning based software company with a mission to open source the law.

We are disrupting the legal industry by changing the business model ground up.

What Github did with open source code, Instagram and TikTok with entertainment, Airbnb with hospitality, and Uber with travel, Genie is doing with Legal contracts.

We are building a community-based AI law platform.



Requirements
About the role
We're looking for an experienced Senior React developer who has a passion for building an elegant & modern interface to solve complex real-world problems.

You are a product-focused, design thinking and curious developer who wants to turn user needs into simple yet clever solutions
You are excited to develop a community-based platform web application in React
You are interested in creating a rich, beautiful and elegant user interface to perfection
You enjoy and find it fun to build re-usable components
You are keen to protect your code with solid tests ranging from unit to automation
You care to collaborate and participate in solution design and performing experiments with the team
You are not afraid to take risks and learn advanced topics
You are eager for friendly, coherent spoken and written communication to exchange complex ideas
You build coding standards and peer review code to ensure code is neat and clean
What we need
Experience in building SaaS web-based products using;

Solid knowledge of React or NextJS framework
Typescript and Javascript
Proficient in HTML, CSS
What we would like
GraphQL
Knowledge of Design Patterns in Web development
Experience with test frameworks for unit, integration, and automation testing

What would be nice

Working with real-time collaboration applications & browser based editors
Experience with server-side rendering, web workers & service workers
Built apps with Micro-services architecture
Solid experience in responsive UI development

About our stack
Micro-services architecture, React, Next.js, Angular 12, Playwright, NodeJS, RabbitMQ, GraphQL, Redis, MongoDB, Python, AWS, Docker, Kubernetes, TensorFlow, PyTorch, ProseMirror. You're not expected to know all of this, but you will get the opportunity to learn based on your interest!


Benefits

What do Genies get?
Substantial equipment budget for you to set up a home working environment, including a top of the range laptop, monitor, wireless keyboard, mouse, and office chair
Generous stock options - we want everyone to share in our success!
Unlimited budget for business, law, or technology books
Unlimited holiday
Fully flexible working
Cycle to work scheme
Regular meditation and wellbeing sessions
Employee benefits platform where you can buy plants, food deliveries, yoga, wellbeing sessions, and more on us!
Optional pre-tax charity giving
Regular team lunches and dinners after lockdown (and plenty of social activity in the meantime!)
The opportunity to join a bright, fast-growing startup at a grassroots level and genuinely make a difference

About Genie
Genie AI was created at Entrepreneur First in November 2017. We're now a team of 16 incredibly talented and fun Genies from 7 countries split across Business and Technology.

We’re currently flexible on work from home versus being in the office; our long-term plan is for everyone to work 2 days a week in the office, and the rest from home, so we get the best of both worlds. Our office is based in Shoreditch, London, with lots of great places to eat and drink nearby.

We are backed by world-leading experts such as Lord Neuberger (former President of the Supreme Court), Charlie Songhurst (former Head of Strategy at Microsoft), Khosla Ventures (founded by Vinod Khosla, co-founder of Sun Microsystems, the inventors of Java), Connect Ventures (also funded Citymapper and Typeform) and other investors with serious pedigree.

We have been featured on [TechCrunch](https://techcrunch.com/2019/07/23/genie-ai/) and on [Forbes](https://www.forbes.com/sites/mirunagirtu/2019/09/09/29-of-the-uks-most-exciting-ai-startups-race-to-scale/?sh=26fe97d2a6dc) as one of the top 29 UK AI startups.

We're poised for rapid expansion, and we want you to be our next Genie. Come and shake up the £100bn global legal industry with us!

**Reply by Gyekye** [reply]
The link is not working please

**Reply by Charlette123** [reply]
Hi Richard, thanks for getting in touch! Please feel free to apply here: https://apply.workable.com/genie-ai/j/8166DCED3C/

**Comment by flavoflav** [comment]
[Chapter Three i](http://www.chapterthree.com)s seeking next.js developers to join our team.  This position is for an engineer who has experience working with Decoupled Drupal sites and next.js or any react library. 

Chapter Three is a leader in Next-Drupal integrations and is building our engineering team to build enterprise-class solutions. 

Job is posted on LinkedIn [https://www.linkedin.com/jobs/view/2921352437/](https://www.linkedin.com/jobs/view/2921352437/) or you can email us at jobs@chapterthree.com

You can learn more about Chapter Three here [https://www.chapterthree.com/about](https://www.chapterthree.com/about)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by Sha-code** [comment]
[Danae](https://danae.io/) | Frontend Engineer | Remote (EU) or Paris | Full-time | [Link](https://abyssinian-armadillo-d59.notion.site/Front-end-D-veloppeur-se-Nextjs-Tailwind-0f977c7c935c4f3fb03eda3e48c5197c)
Danae is a tech startup founded by art historians. The goal? Embed digital art in art history and enable art galleries to adopt the NFT model.
### **⭐The missions are:**

👉 Improve, maintain and develop new features on Danae’s front-end

👉 Code your solution and verify that it works within the project

👉 Work on the construction of the Saas Danae

👉 Complete backlog (next tasks to be completed)

👉 Audit the application and be a force for proposal to consolidate security and scalability

## **Profile**

**About You:**

- An appetite for the Front-end (Nextjs/Tailwind), you are interested in the Serverless.
- Well-being and good communication are key to your success
- Managing a young project is not a problem for you
- Want to improve yourself technically. Our project use Hedera hasgraph as Blockchain
- Already worked on a blockchain project and/or Saas is a real plus

Here is also our complete stack: NodeJS, NextJS, tailwind, PostgreSQL, AWS, Github.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by Danm72** [comment]
Mawla - Digital Agency | Mid/Senior React/Jamstack developer | EU REMOTE | Permanent | Minimum €40,000 | Profit share DOE | [Link](https://apply.workable.com/mawla-team/j/A10FE4F0E3)

Delighted to be back here again to hire, Sanity is core to how we work, and we love this community of makers!

We are Mawla, we build incredible digital experience for global SaaS companies.

We are looking to hire a Mid or Senior front-end/Jamstack developer to join the Mawla team. You'll have an eye for detail and be proactive in nature. Strong team communication skills are a must, as is the ability to work autonomously.

What we do
We've built a repeatable engine we're currently calling 'The Mawla Engine', it allows us to start projects at 70% completion and focus on what truly matters to the customer.
Build custom high-performance websites and web apps, primarily using the Jamstack architecture (Next, Sanity, Typescript, Tailwind)
Create rich interactive/animated landing pages

In a typical project, we use:

- React / NextJS
- TailwindCSS for styling
- Framer motion for animation
- Serverless functions
- Sanity (CMS)
- Typescript

Please DM me for more info, and apply through [here](https://apply.workable.com/mawla-team/j/A10FE4F0E3/)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by dfeldman0** [comment]
Excited to share an [opportunity as the **technical lead our NextJS** / Jamstack web team in the Boston area.  ](https://takeda.wd3.myworkdayjobs.com/External/job/Lexington-MA/Jamstack-Development-Lead_R0058401-1)
Very competitive base salary plus annual and long-term incentive bonuses.

At Takeda, we are transforming the pharmaceutical industry through our R&D-driven market leadership and being a values-led company. To do this, we empower our people to realize their potential through life-changing work. Certified as a Global Top Employer, we offer stimulating careers, encourage innovation, and strive for excellence in everything we do. We foster an inclusive, collaborative workplace, in which our global teams are united by an unwavering commitment to deliver Better Health and a Brighter Future to people around the world.

- Lead the development of templates and self-service capabilities to enable the business to rapidly create flexible websites using NextJS
- Deep development experience on ReactJS, Next.js with Headless CMS (ex. Sanity, Contentful, ContentStack)
- Experience integrating headless CMS, static site generators, and rich site editor (ex Stackbit)
- Typescript & ES6 Ninja
- Client-side security with OAuth / OIDC- 
- Experience with search in Jamstack architectures (ex. Algolia, Elastic)
- Expert Performance and SEO optimization expertise
- Hands on experience with dynamic personalization and behavior-based persona detection
- Jamstack DevOps experience (ex. Cloudflare Pages, Netlify, Vercel)
- Experience with Edge APIs (ex Cloudfare Workers)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for March is up (https://github.com/vercel/next.js/discussions/34949) and you're more than welcome to repost this message there :)

**Comment by yui-box** [comment]
The Owl Solutions| UX/UI Designer | Remote (US, Canada, or ET timezone) | Full-time | Send us your salary expectation | [Link](https://www.linkedin.com/jobs/view/2940169670/)

In today’s world, supply chain management cannot be a manual process. The more forward-thinking, data-driven your company is, the more money you can keep inside your business. Our co-founders both have decades of experience in supply chain management, and through their journey they realized that companies do not fully capitalize on their available data and therefore they miss multi-million-dollar opportunities to operate more efficiently and effectively.

We developed OWL, an enterprise SaaS software solution that helps manufacturing and distribution companies see their supply chains clearly.

About the Role:

We are seeking a UI/UX Designer to join our Engineering Team and be responsible for designing the best user experience for our SaaS product. This position will be working together with the Senior Engineers and Product Owner.
Please see more details about the position on the link above.
